SourceKitStressTester/Sources/StressTester/ActionGenerators.swift

```diff
@@ -21,13 +21,16 @@ public protocol ActionGenerator {
 extension ActionGenerator {
   /// Entrypoint intended for testing purposes only
   public func generate(for file: URL) -> [Action] {
-    let tree = try! SyntaxParser.parse(file)
-    return generate(for: tree)
+    let fileData = try! Data(contentsOf: file)
```

Comment:
I imagine parsing a file will be a very common case. Perhaps we should add a `.parse` for it? We already do this twice just here 😅

Reply:
I think the reason why we don’t have it is that the parser would then need to depend on Foundation for `URL` and to read the file contents. 😬

Reply:
Sigh. Fair enough. Could add an extension in swift-stress-tester though 🤷